ghost/core/core/server/services/auth/setup.js

```diff
@@ -183,7 +183,7 @@ function sendWelcomeEmail(email, mailAPI) {
                     }]
                 };
 
-                mailAPI.send(payload, {context: {internal: true}})
```

Comment:
Waiting on confirmation from @ErisDS if we can simply make the mail API call synchronous with the rest of the Authentication API. 

If not, I'll revert this line change and make the necessary changes to the mail mocking library.

Reply:
I think we can change this to sync and see if that creates other issues 👍